packages/jsii-pacmak/lib/targets/go.ts

```diff
@@ -54,7 +54,13 @@ export class Golang extends Target {
       );
       return Promise.reject(e);
     }
-    await go('build', ['-modfile', localGoMod.path, './...'], { cwd: pkgDir });
+
+    if (process.env.JSII_GO_DISABLE_BUILD !== '1') {
```

Comment:
Let's make this opt-in... it's a win for everyone... So say if the `JSII_BUILD_GO` variable is set (whatever the value), then we'll run the `go build` step.

```suggestion
    if (process.env.JSII_BUILD_GO) {
```

Reply:
That’s ok for me as well 👍